crates/core/src/shared/transform.rs

```diff
@@ -205,8 +272,117 @@ impl<'a> TransformResult<'a> {
         match config.generate {
             OutputType::Dom => {
                 // TODO
+                // if template is not null, create template => import { template as _$template } from "solid-js/web"; ... (client side rendering)
                 ctx.ast.expression_null_literal(SPAN)
             }
         }
     }
 }
+
+#[cfg(test)]
+mod transform_tests {
+    use super::*;
+    use oxc::{allocator::Allocator, parser::Parser, semantic::SemanticBuilder, span::SourceType};
+
+    struct test_case {
```

Comment:
Clippy wouldn't like the casing 😅 (I'll set it up in the CI soon)

Reply:
clippy would love this fix https://github.com/XiNiHa/oxc-jsx-dom-expressions/pull/2/commits/3c91e3bcfb8aa9e314669b03270dfa18a2ca2e0b